chipsec/hal/uefi_platform.py

```diff
@@ -521,17 +521,17 @@ def __str__(self):
         return """
 Header (VSS)
 ------------
-VendorGuid : {%08X-%04X-%04X-%04s-%06s}
+VendorGuid : {%s}
```

Comment:
As part of this change is you have fixed some print formatting and left other formatting.  Is this on purpose?

Reply:
The fixes were to help simplify some of the guid parsing and make it python 2/3 compliant.  Probably should have put them in a separate pull request...